flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/HiveTableSource.java

```diff
@@ -257,6 +279,12 @@ public void applyProjection(int[][] projectedFields, DataType producedDataType)
         this.projectedFields = Arrays.stream(projectedFields).mapToInt(value -> value[0]).toArray();
     }
 
+    @Override
+    public Result applyFilters(List<ResolvedExpression> filters) {
+        this.filters = new ArrayList<>(filters);
+        return Result.of(new ArrayList<>(filters), new ArrayList<>(filters));
```

Comment:
Is this unimplemented to support orc filter pushdown?

Reply:
or now we just pushdown the orc filter to orc input format, but at planner level, we keep all the filter above the source.

Reply:
Yes, you're right. We just pushdown the orc filter to orc input format. 
The reason is some formats don't implement filter pushdown or don't support filter pushdown. So, we still make the plan keep all the filters, which can make sure the result is correct.
I think i'll be better to prun  hese filtres in here, but it's hard/impossible to do that.

Reply:
@luoyuxia Thanks for reply. I also want to know what's the hard part, if the filter can pushed to orc, maybe we can remove it from the original filters, it may be more reasonable.

Reply:
What if the underlying format is text? If we remove it from the original filters, it will bring wrong result since text format don't support filter pushdown . The hard is we can't know what's the underlying format in plan phase currently though we can propose some new interfaces.

Reply:
@luoyuxia what if to add a method to indenify format in HiveTableSource, and when implement applyFilter, we can get this format info, so to better implement applyFilter?

Reply:
I don't think indenify format in HiveTableSource is possible considering different partitions may have different formats and we don't know which partitions it expect to read.

Reply:
@luoyuxia I agree with you.